app/models/mentor.rb

```diff
@@ -0,0 +1,25 @@
+class Mentor < ApplicationRecord
+  attr_reader :name,
+  :email,
```

Comment:
I recommend giving a uniqueness constraint to the email attribute

Reply:
Good call.